docs/example.md

```diff
@@ -2,34 +2,47 @@
 title: Connector Example
 ---
 
-A connector is a set of files that describe the UI elements needed to collect user input for creating a connection to a data source, any dialect or customizations needed, a connection string builder, driver resolver, and the ODBC- or JDBC-based driver.
-Starting with the set of base connector files, you can add customizations to each file, while using the connectivity test harness to validate the connector behavior along the way.
-The base connector files are described below.
+A Tableau connector is a set of files that describe the UI elements needed to collect user input for creating a connection to a data source, any dialect or customizations needed, a connection string builder, driver resolver, and the ODBC- or JDBC-based driver.
 
-This example will use the PostGres sample connector located in the [postgres_odbc or postgres_jdbc folder](https://github.com/tableau/connector-plugin-sdk/tree/master/samples/plugins) which is a connector for the PostGre SQL database.
+Tableau connector files include:
+- __Manifest file__ that tells Tableau about the connector. Tableau uses the name from this file to add it to the list of available connectors in the UI.
+- __Tableau Custom Dialog file__ is optional. It customizes the connection dialog as needed for this connection. 
```

Comment:
If `odbc` or `jdbc` are chosen as the base for a plugin, which hopefully is the common case, then the tcd, tdr and tdd files actually aren't optional, so it might not be ideal to emphasis that here.